package.json

```diff
@@ -38,63 +38,63 @@
     "vendor/**/*"
   ],
   "dependencies": {
-    "custom-event-polyfill": "^1.0.6",
+    "custom-event-polyfill": "^1.0.7",
```

Comment:
Better to not touch dependencies. Only upgrade when necessary.

Reply:
I agree; and also, if package.json is updated, it will likely require moving forward the aframe version number. That said, if you use a clean machine and git clone the repo, you will find a number of issues with the current package.json file that makes updating pieces of it problematic. Can anyone else verify with a clean machine that package.json is stable? Thanks!

Reply:
Travis (our CI) machine does that per commit and haven't heard of any issues. Better to revert the package.json file to the original version and address any issues in a seperate PR

Reply:
I've reverted the package.json and resolved all the other issues described. The big remaining question is, do we want to have a temporary or permanent visible icon on the bottom left, the way it is now; or, do we want to have a full screen interrupt, similar to 8thwall or vr-mode-ui when you are in portrait mode, to question device-motion-permission? I believe it is ready to be checked in.

Reply:
Thanks for the hard work. I would say we want a modal dialog similar to 8thwall or vr-mode-ui. Button is easy to miss and hard to understand what it's for. We can apply and reuse vr-mode-ui  style for consistency.